lib/SPIRV/libSPIRV/SPIRVEntry.cpp

```diff
@@ -302,8 +302,7 @@ void SPIRVEntry::setLine(const std::shared_ptr<const SPIRVLine> &L) {
 }
 
 void SPIRVEntry::addMemberDecorate(SPIRVMemberDecorate *Dec) {
-  assert(canHaveMemberDecorates() &&
-         MemberDecorates.find(Dec->getPair()) == MemberDecorates.end());
+  assert(canHaveMemberDecorates());
```

Comment:
Should we allow duplication only if it's UserSemantic?

Reply:
There are might be other decorations that are allowed to be duplicated. So assertion as an assumption that it never happens seems to be incorrect.